docs/yul.rst

```diff
@@ -1248,7 +1248,7 @@ You can override this sequence and supply your own using the ``--yul-optimizatio
 
 .. code-block:: sh
 
-    solc --optimize --ir-optimized --yul-optimizations 'dhfoD[xarrscLMcCTU]uljmul'
+    solc --optimize --ir-optimized --yul-optimizations 'dhfoD[xarrscLMcCTU]uljmul:fDnTOc'
```

Comment:
I think we should either keep that as before and merely *mention* the delimiter below - or we need to rephrase the paragraph below.

If we keep the delimiter in the example, below should probably rather be something like:

```
The colon delimiter (``:``) in the example is used to specify a custom cleanup sequence, which is run after the main part of the sequence before the delimiter (and, when using the legacy EVM code transform, only *after* the stack compressor). If no such delimiter is present, the default cleanup sequence (``fDnTOc``) is run after the supplied sequence.
```

or so...

Reply:
I'd like to keep the delimiter in the example, since it's now a part of the optimization sequence. I don't really see what the rephrasing actually changes in comparison to the current wording; is it just for style?

Reply:
Without the change the phrasing is contrary to the example...
Then the example *does* have the delimiter and *does not* run any kind of default sequence and only the case that is *not* shown, in which there was *no* delimiter would run a default sequence. That's the difference.
Having the delimiter in the example and still just starting off with the behaviour as if it *wasn't* there, is confusing.